scan_matcher:
  ros__parameters:
    input_cloud_topic: "/input_cloud"
    output_cloud_topic: "/modified_cloud"
    global_frame_id: "map"
    registration_method: "NDT"
    ndt_resolution: 2.0
    ndt_num_threads: 2
    gicp_corr_dist_threshold: 5.0
    trans_for_map_update: 0.5
    rotation_for_map_update: 0.2
    vg_size_for_input: 0.1
    vg_size_for_map: 0.1
    use_min_max_filter: false
    scan_min_range: 10.0
    scan_max_range: 120.0
    scan_period: 0.2
    map_publish_period: 1.0
    num_targeted_cloud: 20
    debug: true
    debug_cloud_topic: "/target_cloud"

multi_session_slam:
  ros__parameters:
    input_cloud_topic: "/modified_cloud"
    output_cloud_topic: "/output_cloud"
    session_start_service: "/session_start"
    session_end_service: "/session_end"
    global_frame_id: "map"
    vg_size_for_input: 0.1

    registration_method: "NDT"
    voxel_leaf_size: 0.2
    threshold_loop_closure_score: 1.0
    range_of_searching_loop_closure: 20.0
    loop_closure_search_num: 10.0
    num_adjacent_pose_constraints: 5
